src/main/kotlin/calculator/operation/NumberOperation.kt

```diff
@@ -0,0 +1,12 @@
+package calculator.operation
+
+import java.math.BigDecimal
+
+class NumberOperation(value: String) : InfixOperation {
+    private val value: BigDecimal
+    init {
+        require(value.matches("[0-9]+".toRegex())) { throw IllegalArgumentException("$value 는 숫자여야합니다.") }
```

Comment:
```suggestion
        require(value.matches("[0-9]+".toRegex())) { "$value 는 숫자여야합니다." }
```
require문의 람다에는 주로 에러 메시지를 작성합니다.